nengo/builder/probe.py

```diff
@@ -64,7 +64,7 @@ def signal_probe(model, key, probe):
 
 probemap = {
     Ensemble: {"decoded_output": None, "input": "in", "scaled_encoders": "encoders"},
-    Neurons: {"output": None, "spikes": None, "rates": None, "input": "in"},
+    Neurons: {"output": None, "input": "in"},
```

Comment:
I don't think it was ever possible to do `Probe(ensemble, "spikes")`; I think it's always had to be `Probe(ensemble.neurons, "spikes")` (at least since we introduced `Neurons`, and who knows when that was).

Reply:
Yeah it might have been a very long time ago :dizzy_face:

Reply:
Yeah that's a good point about backwards compatibility though. I think if we changed it to `{"output": None, "spikes": "out", "rates": "out", "input": "in"}` then it'd all work as expected right? And then sidenote while I was thinking about that, is there a reason to do `{"output": None}` instead of `{"output": "out"}`? The former will create a connection probe rather than a signal probe, but since we're not doing any decoding or anything on that connection it just seems like unnecessary extra work. The same would be true for `Node` I think. But it seems like we probably had a reason for doing it this way?

Reply:
That's a good question as to why we use a connection probe instead of a signal probe. I don't see the connection probe offering any additional benefits in this case, so I think it makes sense to go with the signal probe.

Reply:
I tested out the signal probe change, and it seemed to work, so I've pushed that here.

Reply:
Sounds good, I can't think of a reason why we used a connection probe. I'll add in the `"spikes"` and `"rates"` just for good measure. I would be surprised if we don't rework probes in the builder refactoring anyway, so I'm not going to add a TODO here to take them out as they'll likely be taken out during that backwards-incompatible change anyhow.